src/main/resources/templates/courses/assignments/group-assignment-view.ftl

```diff
@@ -0,0 +1,169 @@
+        <div class="col-md-2">
+        [@projectFrameset.renderSidemenu "assignments" i18n group repository/]
+        </div>
+        <div class="col-md-10">
+            [#if assignment.getSummary()??]
+                <p>${assignment.getSummary()}</p>
+            [/#if]
+
+            [#if assignment.getDueDate()??]
+                <p>
+                    <strong>Due date</strong>
+                    <span>${assignment.getDueDate()}</span>
+                </p>
+            [/#if]
+
+            <table class="table table-bordered">
+            [#if myDeliveries?? && myDeliveries?has_content]
+                [#list myDeliveries as delivery]
+                    [#assign commitId = delivery.getCommitId()!]
+                    [@commitRow.render group states commitId! ""]
+                        [@deliveryStateButton user course delivery/]
+
+                        <div class="committer">${delivery.createdUser.getName()} on ${delivery.getCreated()?string["EEEE dd MMMM yyyy HH:mm"]}</div>
+
+                        [#if delivery.getNotes()??]
+                            <p>${delivery.getNotes()}</p>
+                        [/#if]
+
+                        [#assign attachments = delivery.getAttachments()!]
+                        [#if attachments?has_content]
+                            <ul class="list-inline">
+                            [#list attachments as attachment]
+                                <li>
+                                <a class="btn btn-link btn-sm" target="_blank" href="/courses/${group.course.getCode()}/groups/${group.getGroupNumber()}/assignments/${assignment.getAssignmentId()}/attachment/${attachment.getPath()?url('ISO-8859-1')}">
+                                    <span class="glyphicon glyphicon-file aria-hidden="true"></span>
+                                ${attachment.getFileName()}
+                                </a>
+                                </li>
+                            [/#list]
+                            </ul>
+                        [/#if]
+
+                        [#assign review = delivery.getReview()!]
+                        [#if review?? && review?has_content && (review.getGrade()?? || review.getCommentary()??)]
+                            <blockquote>
+                                <dl>
+                                    [#if review.getGrade()??]
+                                        <dt>Grade</dt>
+                                        <dd>${review.getGrade()}</dd>
+                                    [/#if]
+                                    [#if review.getCommentary()??]
+                                        <dt>Remarks</dt>
+                                        <dd>${review.getCommentary()}</dd>
+                                    [/#if]
+                                </dl>
+                                <footer class="small">${review.reviewUser.getName()} on ${review.getReviewTime()?string["EEEE dd MMMM yyyy HH:mm"]}</footer>
```

Comment:
It would be nice if there is a mechanism to contact your reviewer for explanation or if the group thinks the review is incorrect. A possible way could be to be able to add comments to this, or you can also create the mail button like the assistants overview on the course overview page for assistants themselves. (https://github.com/devhub-tud/devhub/pull/81/files#diff-f772ff623cca82e41932d9628e815d37R125)